doc/nrf/libraries/bluetooth_services/mesh/sensor.rst

```diff
@@ -31,10 +41,17 @@ A sensor reporting the device operating temperature could combine the Bluetooth
 
    static int temp_get(struct bt_mesh_sensor *sensor,
                        struct bt_mesh_msg_ctx *ctx,
-                       struct sensor_value *rsp)
+                       struct bt_mesh_sensor_value *rsp)
    {
+       struct sensor_value value;
```

Comment:
bt_mesh_sensor_value?

Reply:
Nope. This is the value coming from Zephyr's sensor API (as in, the API for getting data from physical sensors). This being `sensor_value` is the entire reason why `bt_mesh_sensor_value_from_sensor_value` exists.